app/models/names_manager/hard_coded_authors.rb

```diff
@@ -83,6 +83,13 @@ def hard_coded_authors(commit)
         # This patch comes from https://github.com/rails/web-console/pull/91,
         # originally authored by Daniel, but ported to upstream Rails by Genadi.
         ['Daniel Rikowski', 'Genadi Samokovarov']
+      when '92209356c310caabda8665d6369d3b1e4a1800d1'
+        # Tsukuru Tanimichi originall worked on it but Aaron Patterson and
+        # Eileen Uchitelle changed the implementation on a separate branch
+        ['Eileen M. Uchitelle', 'Aaron Patterson', 'Tsukuru Tanimichi']
+      when '9668cc3bb03740b13477df0832332eec71563bc5'
+        # Backport of the above commit
```

Comment:
Knowing the doc-shark @fxn, he'd probably want periods at the end of your comments 😉